products/sles.md

```diff
@@ -26,111 +26,155 @@ releases:
     eol: true
     extendedSupport: false
 
+    latestReleaseDate: 2023-06-20
```

Comment:
LGTM : we don't want to display latest releases, but still want to be notified of new releases.